tests/tests/util.ts

```diff
@@ -118,34 +128,71 @@ export async function startMoonbeamNode(
     binary.stdout.on("data", onData);
   });
 
+  const wsProvider = new WsProvider(`ws://localhost:${WS_PORT}`);
+  const polkadotApi = await ApiPromise.create({
+    provider: wsProvider,
+    types: {
+      AccountId: "EthereumAccountId",
+      Address: "AccountId",
+      Balance: "u128",
+      RefCount: "u8",
+      // mapping the lookup
+      LookupSource: "AccountId",
+      Account: {
+        nonce: "U256",
+        balance: "u128",
+      },
+      Transaction: {
+        nonce: "U256",
+        action: "String",
+        gas_price: "u64",
+        gas_limit: "u64",
+        value: "U256",
+        input: "Vec<u8>",
+        signature: "Signature",
+      },
+      Signature: {
+        v: "u64",
+        r: "H256",
+        s: "H256",
+      },
+    },
+  });
+
```

Comment:
These should be taken from the dedicated files rather than added here explicitly

Reply:
It will once the polkadotjs types pr is approved